### PR TITLE
Add jensen glasses

### PR DESCRIPTION
**Pull request by MACMAN2003**
## About the PR

Added yellow jensen glasses,
## Why / Balance

Because there's a jensen coat, and a jensen hairstyle, why not have jensen's glasses too..
## Technical details

n/a
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

## Breaking changes

n/a
**Changelog**

:cl:
- add: Added yellow tinted jensen glasses that may show up in maints from time to time

### CODE REVIEW

**Comment by UbaserB** [comment]
the grey corner looks a little odd hm

**Comment by metalgearsloth** [comment]
Remove the hamster variants.

**Comment by dvir001** [comment]
I never asked for this.

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -79,6 +79,20 @@
...
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesJensen
+  name: jensen glasses
+  description: A pair of yellow tinted folding glasses. You never asked for these.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/jensen.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Glasses/jensen.rsi
+  - type: Tag
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:93]: tag component not needed

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Pull request merged**